### PR TITLE
swap tendermint with cometbft v0.37.0

### PR DESCRIPTION
**Pull request by agouin**

### CODE REVIEW

File: .golangci.yml
Comment (agouin) [reply .golangci.yml]: works for me!

**Review by DavidNix — APPROVED** [review]

**Comment by DavidNix** [comment]
For the record, I do think it's risky to switch these critical packages after all the changes we've made thus far to support ICS. We know the old tendermint packages were stable. My preference would be to hold off on this change until we've done a release + testing on testnet. I'm betting we'll be ok (which is why I've approved), but if there are issues, it may be hard to track down the change that introduced a problem. Was it our refactor for ICS, was it cometbft, etc.

**Pull request merged**
